python/src/deltachat/tracker.py

```diff
@@ -20,6 +20,16 @@ def ac_process_ffi_event(self, ffi_event):
         elif ffi_event.name == "DC_EVENT_IMEX_FILE_WRITTEN":
             self._imex_events.put(ffi_event.data2)
 
+    def wait_progress(self, target_progress, progress_upper_limit=1000, progress_timeout=60):
+        while True:
+            ev = self._imex_events.get(timeout=progress_timeout)
+            if isinstance(ev, int) and ev >= target_progress:
+                assert ev <= progress_upper_limit, \
+                    str(ev) + " exceeded upper progress limit " + str(progress_upper_limit)
+                return ev
+            if ev == 1000 or ev == 0:
```

Comment:
The check `ev == 1000` is not needed, this case is handled by the block right above.